Source/cursor.cpp

```diff
@@ -133,19 +140,34 @@ void FreeCursor()
 	ClearCursor();
 }
 
+const CelSprite &GetInvItemSprite(int i)
+{
+	return i < InvItems1Size ? *pCursCels : *pCursCels2;
+}
+
+int GetInvItemFrame(int i)
+{
+	return i < InvItems1Size ? i : i - InvItems1Size;

```

Comment:
This _used_ to be an adjustment of -= 179. Which would actually skip the first entry in the hellfire array portion?

Reply:
Save game for testing https://github.com/diasurgical/devilutionX/files/5373942/hsv.zip

Reply:
Changed and added a leading zero to the array.
The weird thing is that the frame number seems to start at 1 instead of 0

Reply:
@AJenbo Thanks for the test file, seems to work :+1: